src/test/kotlin/com/franzi/todochallenge/controller/TaskControllerIT.kt

```diff
@@ -0,0 +1,54 @@
+package com.franzi.todochallenge.controller
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.jayway.jsonpath.JsonPath
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers.print
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.http.MediaType
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.*
+import java.util.*
+
+
+@SpringBootTest
+@AutoConfigureMockMvc
+internal class TaskControllerIT {
```

Comment:
maybe you can add test for `update` and `delete` as well to here?

Reply:
i will add it later.  i will have a branch for writing those tests. for now i need to push to master because i want to push to heroku